src/arduino.cc/arduino-builder/main.go

```diff
@@ -400,9 +387,11 @@ func printError(err error, printStackTrace bool) {
 func printCompleteError(err error) {
 	err = i18n.WrapError(err)
 	fmt.Fprintln(os.Stderr, err.(*errors.Error).ErrorStack())
+	os.Exit(1)
```

Comment:
This seems unexpected to me - I wouldn't expect a function called "printCompleteError" to exit the current process. I guess the name should be changed? Or can this exit be removed when implementing the exit-related changes discussed elsewhere?